modules/optflow/src/learn_prior.py

```diff
@@ -0,0 +1,123 @@
+import os, sys
```

Comment:
Consider adding file comment, to give some idea on the purpose of this script.

Also suggest putting this through a PEP8 autoformatter, like yapf.